OpenStack-accounting/usr/local/sbin/glance_extract_accounting.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+import accountinglib
+
+import sys
+import time
+import datetime
+
+
+nowtime = time.localtime()
+
+print("Glance Accounting run start")
+starttime=sys.argv[1]
+print("Start Time = " + starttime)
+endtime=sys.argv[2]
+print("End Time = " + endtime)
+endyyyymm=datetime.datetime.strptime(endtime,"%Y-%m-%d %H:%M").strftime('%Y-%m')
+endtimestamp = time.mktime(datetime.datetime.strptime(endtime, "%Y-%m-%d %H:%M").timetuple())
+
+
+results = accountinglib.get_accounting_data("glance",starttime,endtime)
+datastring = ''
+print(results)
+for result in results:
+    print(result)
+    try:
+        if "rally" in result["Project"] :
+            department="STFC Cloud"
+        else:
+            if "efault" in result["Department"]:
+                department=result["Project"]
+            else:
+                department=result["Department"]
+
+    except:
```

Comment:
Suggestion: you should really specify the exception you're looking for 

If you're trying to catch any exception that can be raised - you should at least log what the exception was so we can track it
```
except Exceptions as exp: 
    print("An exception was raised: ", str(exp)
````